main.go

```diff
@@ -37,7 +37,6 @@ func main() {
 	log.Info("Starting CF CockroachDB broker")
 
 	InitServicesAndPlans()
-	defer CleanupPlans()
```

Comment:
Why did you remove this line?  What does it do?

Reply:
That removed the temporary files for holding the CA, cert, and key files which we no longer create